documentation/examples/prometheus-kubernetes.yml

```diff
@@ -28,11 +28,9 @@ scrape_configs:
     # insecure_skip_verify: true
   bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
 
-  kubernetes_sd_configs:
-  - api_servers:
-    - 'https://kubernetes.default.svc'
-    in_cluster: true
-    role: apiserver
+  static_configs:
+  - targets:
+    - kubernetes.default.svc
```

Comment:
Kubernetes registers a default `kubernetes` service with respective endpoints for the API server. Hence, we can scrape it just like any other service/endpoint list.